<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTPcapReplayClass</class>
 <widget class="QMainWindow" name="QTPcapReplayClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QTPcapReplay</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images.png</normaloff>images.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QComboBox" name="IPComboBox"/>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget">
      <property name="movement">
       <enum>QListView::Static</enum>
      </property>
      <property name="resizeMode">
       <enum>QListView::Fixed</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="schedulerStatus">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="StatusReportLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="curPcapPackage">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Speed:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="speed_LineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="startReplayButton">
          <property name="text">
           <string>Play</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pauseReplayButton">
          <property name="text">
           <string>Pause</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopReplayButton">
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeFileButton">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exitAppButton">
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>OffSet:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="offSet_lineEdit"/>
      </item>
      <item>
       <widget class="QCheckBox" name="onlyFirstFile_checkbox">
        <property name="text">
         <string>Only First File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="regularReplayButton">
        <property name="text">
         <string>Regular Replay</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="netSpeedMon">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="packetTimeLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>643</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionFiles"/>
    <addaction name="actionFrom_Folder"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_Config"/>
    <addaction name="actionSave_Config"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuOther">
    <property name="title">
     <string>Other</string>
    </property>
    <addaction name="actionNew_Window"/>
    <addaction name="actionNew_Window_Dynamic"/>
    <addaction name="separator"/>
    <addaction name="actionScheduler"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOther"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFiles"/>
   <addaction name="actionFrom_Folder"/>
   <addaction name="actionSave_Config"/>
   <addaction name="actionLoad_Config"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFiles">
   <property name="text">
    <string>Open Files...</string>
   </property>
  </action>
  <action name="actionFrom_Folder">
   <property name="text">
    <string>Open Folder...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionNew_Window">
   <property name="text">
    <string>New Window</string>
   </property>
  </action>
  <action name="actionNew_Window_Dynamic">
   <property name="text">
    <string>New Window Dynamic</string>
   </property>
  </action>
  <action name="actionSchduler">
   <property name="text">
    <string>Schduler</string>
   </property>
  </action>
  <action name="actionScheduler">
   <property name="text">
    <string>Scheduler</string>
   </property>
  </action>
  <action name="actionSave_Config">
   <property name="text">
    <string>Save Config</string>
   </property>
  </action>
  <action name="actionLoad_Config">
   <property name="text">
    <string>Load Config</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qtpcapreplay.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>onFileOpen()</slot>
  <slot>onFolderOpen()</slot>
  <slot>onFileExit()</slot>
  <slot>onItemActivated()</slot>
  <slot>removeFile()</slot>
 </slots>
</ui>
